docs/UserGuide.adoc

```diff
@@ -57,8 +57,13 @@ e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
 [[Features]]
 == Features
 
+=======
 This section describes each command in ContactSails in detail.
 
+=== "Person Panel for v1.5.."
+
+This panel will show the contact details when the contact is selected.
+The feature is to show contact details and additionally, their photo.
```

Comment:
Are you sure this should be here? Because the line above says `This section describes each command in ContactSails in detail`.

Reply:
Plus, the formatting of the headers doesn't seem to work after this change was added...